Aptoide/bds-sdk
# Appc 1898 use product label instead of sku on purchase flow

**What does this PR do?**

   Uses the product label on item's UI references  instead of using the sku.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppcoinsBillingStubHelper.java
- [ ] BuyItemProperties.java
- [ ] PaymentMethodsFragmentLayout.java

**How should this be manually tested?**

Run the toolbox and change the lauchbillingFlow sku from gas to escapedchar.
Confirm that the layout label that appears is Escaped Char

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1898](https://aptoide.atlassian.net/browse/APPC-1898)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1898](https://aptoide.atlassian.net/browse/APPC-1898)

**Questions:**

No questions

**Code Review Checklist**
